ui/pages/routes/routes.component.js

```diff
@@ -353,7 +357,14 @@ export default class Routes extends Component {
         )}
         <NetworkDropdown />
         <AccountMenu />
-        <div className="main-container-wrapper">
+        <div
+          // className="main-container-wrapper"
+          className={
+            classnames('main-container-wrapper', {
+              'main-container-wrapper--filter': isAccountMenuOpen && getEnvironmentType() != ENVIRONMENT_TYPE_FULLSCREEN,
+              'main-container-wrapper--filter': networkDropdownOpen && getEnvironmentType() != ENVIRONMENT_TYPE_FULLSCREEN,
```

Comment:
- Can this be combined into
    `'main-container-wrapper--filter': (isAccountMenuOpen || networkDropdownOpen) && getEnvironmentType() != ENVIRONMENT_TYPE_FULLSCREEN` ?

Reply:
fixed in a176d61bfec30de510c149b1d6e22d9b6619a283